### PR TITLE
Update platform to qemu for all targets

### PR DESCRIPTION
**Pull request by gabrielmocanu**
Update targets to qemu and modify the order for all libraries.

### CODE REVIEW

**Review by craciunoiuc — APPROVED** [review]
All good here. Thanks!

Reviewed-by: Cezar Craciunoiu <cezar.craciunoiu@unikraft.io>

**Review by razvand — APPROVED** [review]
Reviewed-by: Razvan Deaconescu <razvand@unikraft.io>
Approved-by: Razvan Deaconescu <razvand@unikraft.io>

File: kraft.yaml [hunk 1]
```diff
@@ -10,19 +10,21 @@ targets:
...
   - architecture: x86_64
-    platform: kvm
+    platform: qemu
   - architecture: x86_64
     platform: xen
   - architecture: arm64
-    platform: kvm
+    platform: qemu
+  - architecture: x86_64
+    platform: firecraker
```
Comment (nderjung) [inline kraft.yaml:19]:
```diff
- platform: firecracker
+ platform: fc
```

**Review by nderjung — CHANGES_REQUESTED** [review]
~~See inline~~ KraftKit has been updated to handle the fc/firecracker alias variants

**Comment by razvand** [comment]
@nderjung , what needs to happen here? Use `fc` instead of `firecracker`?

**Comment by gabrielmocanu** [comment]
> @nderjung , what needs to happen here? Use `fc` instead of `firecracker`?

Both are accepted, you can see this [here](https://github.com/unikraft/kraftkit/blob/57e76ddf6376f40e1f541e95fe9f6a9c12059c1c/unikraft/plat/platform.go#L96).

**Review by razvand — APPROVED** [review]
Reviewed-by: Razvan Deaconescu <razvand@unikraft.io>
Approved-by: Razvan Deaconescu <razvand@unikraft.io>

**Pull request closed without merging**